pages/models.py

```diff
@@ -548,64 +548,197 @@ def get_context(self, *args, **kwargs):
         ]
 
 
-
-## CANDIDATES
-
+# CANDIDATES
 class CandidateEndorsementPage(Page):
-    body = RichTextField()
-    candidate = models.ForeignKey('endorsements.Candidate', null=True, blank=True, on_delete=models.SET_NULL)
+    result_choices = (
+        ('win', 'Win'),
+        ('loss', 'Loss'),
+    )
+    result_max_length = 16
+
+    body = RichTextField(verbose_name="Bio")
+    candidate = models.ForeignKey(
+        'endorsements.Candidate',
+        help_text='Ignore - legacy field for old pages.',
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL
+    )
+    donate_url = models.URLField(null=True, blank=True)
     election = models.ForeignKey(
         'endorsements.Election',
         null=True,
         blank=True,
         on_delete=models.SET_NULL
     )
-    signup_tagline = models.CharField(max_length=128, blank=True, null=True)
-    social_image = models.ForeignKey('wagtailimages.Image', null=True, blank=True, on_delete=models.SET_NULL, related_name='+')
-    parent_page_types = ['pages.CandidateEndorsementIndexPage']
+    facebook_url = models.URLField(null=True, blank=True)
+    general_election_date = models.DateField(null=True, blank=True)
+    general_election_result = models.CharField(
+        max_length=result_max_length,
+        choices=result_choices,
+        null=True,
+        blank=True
+    )
+    instagram_url = models.URLField(null=True, blank=True)
+    office = models.CharField(null=True, blank=True, max_length=128)
+    photo = models.ForeignKey(
+        'wagtailimages.Image',
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name='+'
+    )
+    primary_election_date = models.DateField(null=True, blank=True)
+    primary_election_result = models.CharField(
+        max_length=result_max_length,
+        choices=result_choices,
+        null=True,
+        blank=True
+    )
+    social_image = models.ForeignKey(
+        'wagtailimages.Image',
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name='+'
+    )
+    state_or_territory = models.CharField(null=True, blank=True, max_length=32)
+    twitter_url = models.URLField(null=True, blank=True)
+    volunteer_url = models.URLField(null=True, blank=True)
+    website_url = models.URLField(null=True, blank=True)
+    youtube_url = models.URLField(null=True, blank=True)
 
+    parent_page_types = ['pages.CandidateEndorsementIndexPage']
 
     content_panels = Page.content_panels + [
-            FieldPanel('body', classname="full"),
-            FieldPanel('candidate'),
-            FieldPanel('election')
-        ]
+        MultiFieldPanel(
```

Comment:
I really like the `MultiFieldPanel` usage here and think this is really intuitive UX to group them in this way. No action needed here.